entrypoint.sh

```diff
@@ -122,6 +118,16 @@ else
 fi
 echo ::endgroup::
 
+echo ::group::Code check
+make codecheck 2>&1
```

Comment:
I think we may need to check whether the `codecheck` command is available here, for libraries not depending on `ign-cmake`, like `ign-tools`. The other one I can think of is `sdformat`, but that's not using this action right now (https://github.com/osrf/sdformat/pull/266).

Reply:
Good catch! I went ahead and added some logic to run `make codecheck` if possible, but to default to `tools/code_check.sh` if `make codecheck` isn't an option: https://github.com/ignition-tooling/action-ignition-ci/blob/5047ff2715648a952c8644906fa526f4e4fb4556/entrypoint.sh#L121-L136